### PR TITLE
Feat: port old packet system over and remove redundant packet(s)

### PR DESCRIPTION
**Pull request by Ilentian**
Ported the old packets over to Google's Protocol Buffer, and removed redundant packets.
Also added comments to more obscure packets.

### CODE REVIEW

File: packets/c2s/ClientHandshakePacket.proto
Comment (cire3wastaken) [inline packets/c2s/ClientHandshakePacket.proto:1]: i guess this is fine

File: packets/s2c/TotemPoppedPacket.proto
Comment (cire3wastaken) [inline packets/s2c/TotemPoppedPacket.proto:1]: you forgot the semicolon at the end of `syntax = "proto3"`

**Review by cire3wastaken — CHANGES_REQUESTED** [review]
fix the syntax error and its approved

**Review by cire3wastaken — APPROVED** [review]
fixed the syntax error, approved

**Comment by cire3wastaken** [comment]
@konekowo opinion before merge?

**Comment by konekowo** [comment]
ready for merge

**Pull request merged**